### PR TITLE
Lm new meta

### PR DESCRIPTION
**Pull request by LouiseMedova**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,6 @@
 .DS_Store
 .vscode
 target/
+ft-storage/.metahash
+ft-logic/.metahash
+ft-main/.metahash
```
Comment (fluiderson) [inline .gitignore:7]:
```suggestion
.metahash
```
A shorter option.

File: ft-storage/io/src/lib.rs [hunk 2]
```diff
@@ -1,7 +1,27 @@
...
+    type State = FTStorageState;
+}
+
+#[derive(Default, Encode, Decode, TypeInfo)]
+pub struct FTStorageState {
+    pub ft_logic_id: ActorId,
+    pub transaction_status: Vec<(H256, bool)>,
+    pub balances: Vec<(ActorId, u128)>,
+    pub approvals: Vec<(ActorId, Vec<(ActorId, u128)>)>,
+}
```
Comment (fluiderson) [inline ft-storage/io/src/lib.rs:23]: Why without:
https://github.com/gear-dapps/sharded-fungible-token/blob/fc36cdb3fe4a8a34e4040ba606176494163b5cf5/ft-storage/src/lib.rs#L15
?

**Review by fluiderson — COMMENTED** [review]
Where is `state` crates?

**Review by bezrazli4n0 — COMMENTED** [review]
Please, fix CI and stay sync with [template Makefile](https://github.com/gear-dapps/app/blob/as-new-meta-rewrite/Makefile)

**Pull request merged**